docs/registry/entities/android.md

```diff
@@ -0,0 +1,20 @@
+<!-- NOTE: THIS FILE IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/entity_namespace.md.j2 -->
+
+# Android
```

Comment:
should we remove convent of https://github.com/open-telemetry/semantic-conventions/blob/main/docs/resource/android.md ? and redirect it to the new page? (and other resource definitions)

Reply:
Done! https://github.com/open-telemetry/semantic-conventions/issues/2327